instrumentation/src/main/java/io/opentelemetry/android/features/diskbuffering/DiskBufferingConfiguration.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.android.config;
+package io.opentelemetry.android.features.diskbuffering;
```

Comment:
It's definitely not needed to reorganize it this way, it's more of a personal preference of mine to have all related things into a single package because I've had issues in the past tracking down related classes that were spread across a project by type. For example, projects with a package named "activities" had all the activities of the app in it, each one representing a feature that needed other classes to work properly but that couldn't be in the same package because they were not of type Activity, and so whenever we needed to do some refactoring or removing some feature we had to go hunting for all the related classes in the project and it wasn't fun 😩 

However, I'm open to suggestions and/or debating what kind of structure could fit better this project! These are the kinds of things that make me want to have a dedicated Android SIG tbh, it seems like the communication would flow a bit faster having that option.

Reply:
Ok, I mean all hierarchal categories are going to fail at some point. :) I don't think this should be blocker here, just mostly wanted to pick your brain.

We could bring up the idea of splitting out a SIG for mobile or even more specifically for Android, but I'd want to be a little cautious and make sure that there's more momentum than just you and me....but really, if we get Hanson and Santosh and Surbhi and @marandaneto interested in a SIG, it could have some value. I do agree that it would help information flow a little faster, especially around larger initiatives and things that are cross-cutting concerns (like package structure, etc).

Let's bring it up at the next client sig.

Reply:
I'm down, my timezone is CET (UTC+1), if I have no collision with the date and time, let's do this.